packages/shared/src/components/cards/common/PostMetadata.tsx

```diff
@@ -49,6 +67,35 @@ export default function PostMetadata({
           <Separator />
         )}
         {!!description && description}
+        {pollHasEnded && (
+          <>
+            <Typography tag={TypographyTag.Span} type={TypographyType.Footnote}>
+              Voting ended
+            </Typography>
+            <Separator />
+          </>
+        )}
+        {isPoll && (
+          <>
+            <Typography
+              tag={TypographyTag.Span}
+              type={TypographyType.Footnote}
+              color={TypographyColor.StatusSuccess}
+            >
+              {shouldShowVotes ? 'Voting open' : 'New poll'}
+            </Typography>
+            <Separator />
+          </>
+        )}
+        {isPoll && shouldShowVotes && (
+          <>
+            <Typography tag={TypographyTag.Span} type={TypographyType.Footnote}>
+              {largeNumberFormat(numPollVotes)}{' '}
+              {pollHasEnded ? 'total votes' : 'votes'}
+            </Typography>
+            <Separator />
+          </>
+        )}
```

Comment:
What if we move those additions to a `PostPollMetadata` component? This will help to keep the component clean. Wdyt?